.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,34 @@
+---
+name: User Story
+about: A template to describe a features or enhancements we want to make from a users
+  perspective to ensure we understand what is needed and the value it would add
+title: ''
+labels: ux & visual design
```

Comment:
```suggestion
labels: ux-and-visual-design
```


I have updated the label to remove spaces to avoid issues with the add issue to project GH Action. THis suggestion aligns to that.